Content.Shared/Actions/SharedActionsSystem.cs

```diff
@@ -1008,6 +1008,16 @@ public void SetIconColor(Entity<ActionComponent?> ent, Color color)
         DirtyField(ent, ent.Comp, nameof(ActionComponent.IconColor));
     }
 
+    // RMC14
+    public void SetItemIconStyle(Entity<ActionComponent?> ent, ItemActionIconStyle style)
+    {
+        if (!_actionQuery.Resolve(ent, ref ent.Comp) || ent.Comp.ItemIconStyle == style)
+            return;
+
+        ent.Comp.ItemIconStyle = style;
+        DirtyField(ent, ent.Comp, nameof(ActionComponent.ItemIconStyle));
+    }
```

Comment:
This should go in an RMC-specific system, if it gives you an access attribute error you can suppress it